app/(admin)/admin/notifications/actions/sendEmailNotification.test.tsx

```diff
@@ -0,0 +1,35 @@
+'use server';
+import { messaging } from "@/api/serverConfig";
+import { sendEmailNotifications } from "./sendEmailNotification";
+
+jest.mock('./sendEmailNotification', () => ({
+    sendEmailNotifications: jest.fn(),
+}))
+
+jest.mock('@/api/serverConfig', () => ({
+    messaging: {
+        createEmail: jest.fn(),
+    },
+}));
+
+describe('SendEmailNotification', () => {
+    it('should send email with provided information', async () => {
+        const content = 'Test';
+        const groupEmailTest = ['123456', '12345', '1234'];
+        const subject = 'This is a test';
+
+        (sendEmailNotifications as jest.Mock).mockImplementation(async ({content, groupEmailTest, subject}) => {
+            await (messaging.createEmail as jest.Mock)('some-id', subject, content, [], groupEmailTest);
+        })
+
+        await sendEmailNotifications({content, groupEmailTest, subject});
+
+        expect(messaging.createEmail).toHaveBeenCalledWith(
+            expect.any(String),
+            subject,
+            content,
+            [],
```

Comment:
Why are we leaving topics as an empty array here?

Reply:
The way the createEmail function is set up is it has a strict order of parameters that get passed through. If I rearrange or even delete one (the empty array) it doesn't know what "emailGroup" is. I have to tell the function that the array is empty